src/foundry/client-esm/applications/apps/file-picker.d.mts

```diff
@@ -0,0 +1,396 @@
+   */
+  static browse(source: string, target: string, options?: FilePicker.BrowseOptions): Promise<FilePicker.BrowseReturn>;
+
+  /**
+   * Configure metadata settings regarding a certain file system path
+   * @param source  - The source location in which to browse: see FilePicker#sources for details.
+   * @param target  - The target within the source location
+   * @param options - Optional arguments modifying the request
+   */
+  static configurePath(
+    source: string,
+    target: string,
+    options?: FilePicker.BrowseOptions,
+  ): Promise<FilePicker.ConfigurePathReturn>;
+
+  /**
+   * Create a subdirectory within a given source. The requested subdirectory path must not already exist.
+   * @param source  - The source location in which to browse: see FilePicker#sources for details.
+   * @param target  - The target within the source location
+   * @param options - Optional arguments which modify the request
+   * @returns The full file path of the created directory
+   */
+  static createDirectory(source: string, target: string, options?: FilePicker.BrowseOptions): Promise<string>;
+
+  /**
+   * Dispatch a POST request to the server containing a directory path and a file to upload
+   * @param source  - The data source to which the file should be uploaded
+   * @param path    - The destination path
+   * @param file    - The File object to upload
+   * @param body    - Additional file upload options sent in the POST body
+   * @param options - Additional options to configure how the method behaves
+   * @returns The response object
+   */
+  static upload(
+    source: string,
+    path: string,
+    file: File,
+    body?: FilePicker.UploadBody,
+    options?: FilePicker.UploadOptions,
+  ): Promise<Response>;
+
+  /**
+   * A convenience function that uploads a file to a given package's persistent /storage/ directory
+   * @param packageId - The id of the package to which the file should be uploaded. Only supports Systems and Modules.
+   * @param path      - The relative destination path in the package's storage directory
+   * @param file      - The File object to upload
+   * @param body      - Additional file upload options sent in the POST body
+   * @param options   - Additional options to configure how the method behaves
+   * @returns The response object
+   */
+  static uploadPersistent(
+    packageId: string,
+    path: string,
+    file: File,
+    body?: FilePicker.UploadBody,
+    options?: FilePicker.UploadOptions,
+  ): Promise<Response>;
+
+  /**
+   * Browse to a specific location for this FilePicker instance
+   * @param target  - The target within the currently active source location.
+   * @param options - Browsing options
+   */
+  browse(target?: string, options?: FilePicker.BrowseOptions): Promise<this>;
+
+  override render(options?: DeepPartial<RenderOptions>): Promise<this>;
+  override render(options: boolean, _options?: DeepPartial<RenderOptions>): Promise<this>;
```

Comment:
Isn't this version of `render` deprecated or something? Will the `@deprecated` be inherited?